app/routes/auth.route.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const router = express.Router();
+const controller = require('../controller/auth.controller.js');
```

Comment:
Codacy has a fix for the issue: [Replace `'../controller/auth.controller.js'` with `"../controller/auth.controller.js"`](https://app.codacy.com/gh/Eight-Corner/oneuleun_web_api/pullRequest?prid=9684192)
```suggestion
const controller = require("../controller/auth.controller.js");
```